benchmarks/Cases/BenchmarkZstdIgbinary.php

```diff
@@ -57,27 +60,40 @@ public function seed(): void
      */
     protected function setSerialization($client): void
     {
-        $client->setOption(Redis::OPT_SERIALIZER, Redis::SERIALIZER_IGBINARY);
-        $client->setOption(Redis::OPT_COMPRESSION, Redis::COMPRESSION_ZSTD);
+        if (! $client->setOption($client::OPT_SERIALIZER, $client::SERIALIZER_IGBINARY)) {
+            Reporter::printWarning(sprintf('Unable to set igbinary serializer on %s', get_class($client)));
+        }
+
+        if (! $client->setOption($client::OPT_COMPRESSION, $client::COMPRESSION_ZSTD)) {
+            Reporter::printWarning(sprintf('Unable to set zstd compression on %s', get_class($client)));
+        }
     }
 
     public function setUpClients(): void
     {
         parent::setUpClients();
 
-        foreach ([
-            $this->phpredis,
+        $clients = [
             $this->relayNoCache,
             $this->relay,
-        ] as $client) {
+        ];
+
+        if (extension_loaded('redis')) {
+            $clients[] = $this->phpredis;
+        }
+
+        foreach ($clients as $client) {
             $this->setSerialization($client);
         }
     }
 
     public function refreshClients(): void
     {
         parent::refreshClients();
-        $this->setSerialization($this->phpredis);
+
+        if (extension_loaded('redis')) {
+            $this->setSerialization($this->phpredis);
+        }
```

Comment:
Is the serialization still set for Relay?

Reply:
Serialization is set in `setUpClients` method